src/lang/modifyAst/addFillet.test.ts

```diff
@@ -229,92 +228,4 @@ const extrude001 = extrude(50, sketch001)
-            |> line([60.04, -55.72], %)
-            |> line([1.29, -115.74], %)
-            |> line([-87.24, -47.08], %, $seg03)
-            |> tangentialArcTo([56.15, -94.58], %)
-            |> tangentialArcTo([14.68, -104.52], %)
-            |> lineTo([profileStartX(%), profileStartY(%)], %)
-            |> close(%)
-          const extrude001 = extrude(50, sketch001)
-            |> fillet({ radius: 10, tags: [seg03] }, %)`
-    const segmentSnippet = `line([-87.24, -47.08], %, $seg03)`
-    const extrudeSnippet = `const extrude001 = extrude(50, sketch001)`
-    const radius = createLiteral(5) as Value
-    const expectedCode = `const sketch001 = startSketchOn('XZ')
-  |> startProfileAt([2.16, 49.67], %)
-  |> line([101.49, 139.93], %)
-  |> line([60.04, -55.72], %)
-  |> line([1.29, -115.74], %)
-  |> line([-87.24, -47.08], %, $seg03)
-  |> tangentialArcTo([56.15, -94.58], %)
-  |> tangentialArcTo([14.68, -104.52], %)
-  |> lineTo([profileStartX(%), profileStartY(%)], %)
-  |> close(%)
-const extrude001 = extrude(50, sketch001)
-  |> fillet({ radius: 5, tags: [seg03] }, %)`
-
-    await runFilletTest(
-      code,
-      segmentSnippet,
-      extrudeSnippet,
-      radius,
-      expectedCode
-    )
-  })
-
-  /**
-   * 6. Case of existing fillet on selected segment
-   */
-
-  it('should modify existing fillet, surrounded with other fillets', async () => {
-    const code = `const sketch001 = startSketchOn('XZ')
-          |> startProfileAt([2.16, 49.67], %)
-          |> line([101.49, 139.93], %)
-          |> line([60.04, -55.72], %, $seg05)
-          |> line([1.29, -115.74], %)
-          |> line([-87.24, -47.08], %, $seg03)
-          |> tangentialArcTo([56.15, -94.58], %)
-          |> tangentialArcTo([14.68, -104.52], %, $seg07)
-          |> lineTo([profileStartX(%), profileStartY(%)], %)
-          |> close(%)
-        const extrude001 = extrude(50, sketch001)
-          |> fillet({ radius: 10, tags: [seg03] }, %)
-          |> fillet({ radius: 15, tags: [seg05] }, %)
-          |> fillet({ radius: 7, tags: [seg07] }, %)`
-    const segmentSnippet = `line([-87.24, -47.08], %, $seg03)`
-    const extrudeSnippet = `const extrude001 = extrude(50, sketch001)`
-    const radius = createLiteral(5) as Value
-    const expectedCode = `const sketch001 = startSketchOn('XZ')
-  |> startProfileAt([2.16, 49.67], %)
-  |> line([101.49, 139.93], %)
-  |> line([60.04, -55.72], %, $seg05)
-  |> line([1.29, -115.74], %)
-  |> line([-87.24, -47.08], %, $seg03)
-  |> tangentialArcTo([56.15, -94.58], %)
-  |> tangentialArcTo([14.68, -104.52], %, $seg07)
-  |> lineTo([profileStartX(%), profileStartY(%)], %)
-  |> close(%)
-const extrude001 = extrude(50, sketch001)
-  |> fillet({ radius: 5, tags: [seg03] }, %)
-  |> fillet({ radius: 15, tags: [seg05] }, %)
-  |> fillet({ radius: 7, tags: [seg07] }, %)`
-
-    await runFilletTest(
-      code,
-      segmentSnippet,
-      extrudeSnippet,
-      radius,
-      expectedCode
-    )
-  })
```

Comment:
I removed these tests because I didn't think they made much sense as the only way to "select" the right source range is for the user to manually put their cursor in the right place, the original edge itself was not selectable since the edge got filleted, so editing the existing fillet I don't think made much sense.

I would however like the fillet itself to be selectable and editable, but that will require some engine updates too.